acceptance/setup/aio/010_Install.rb

```diff
@@ -30,17 +31,12 @@
       on(master, "rpm -Uvh https://yum.puppetlabs.com/puppet5-release-el-#{version}.noarch.rpm")
     end
   else
-    if ENV['SERVER_VERSION'] && ENV['SERVER_VERSION'] != 'latest' && dev_builds_url == 'http://builds.delivery.puppetlabs.net'
+    if ENV['SERVER_VERSION'] && ENV['SERVER_VERSION'] != 'latest' && dev_builds_url == DEFAULT_BUILDS_URL
       install_from_build_data_url('puppetserver', "#{dev_builds_url}/puppetserver/#{ENV['SERVER_VERSION']}/artifacts/#{ENV['SERVER_VERSION']}.yaml", master)
     else
-      if ENV['SERVER_VERSION'].nil? || ENV['SERVER_VERSION'] == 'latest'
-        server_version = ''
-        project = 'puppetserver-latest'
-      else
-        server_version = ENV['SERVER_VERSION']
-        project = 'puppetserver'
-      end
-      install_puppetlabs_dev_repo(master, project, server_version, nil, :dev_builds_url => dev_builds_url)
+      dev_builds_url = 'https://nightlies.puppetlabs.com' if dev_builds_url == DEFAULT_BUILDS_URL
```

Comment:
If `SERVER_VERSION` isn't latest, we still want to enable downloading from builds.delivery.puppetlabs.net

Reply:
if it's not latest (or nil), and we haven't overridden DEV_BUILDS_URL, we would've used line 35 instead.

Reply:
+1